chapter_linear-networks/image-classification-dataset.md

```diff
@@ -201,6 +247,20 @@ train_iter = tf.data.Dataset.from_tensor_slices(
     mnist_train).batch(batch_size).shuffle(len(mnist_train[0]))
 ```
 
+```{.python .input}
+#@tab paddle
+batch_size = 256
+
+def get_dataloader_workers():  #@save
+    """使用4个进程来读取数据"""
+    return 0 if not sys.platform.startswith('linux') else 4
+
+train_iter = paddle.io.DataLoader(dataset=mnist_train,
```

Comment:
I am not sure what happened but it seems the data loading process is much slower than other libraries. It is acceptable though if it cannot be optimized.

Reply:
Hi! 
As the result shown in http://preview.d2l.ai/d2l-zh/PR-1134/, I noticed that a line in the log is 
```
/home/d2l-worker/miniconda3/envs/d2l-zh-paddle-1/lib/python3.8/site-packages/paddle/fluid/framework.py:311: UserWarning: You are using GPU version Paddle, but your CUDA device is not set properly. CPU device will be used by default.
```
Maybe that is why you observed the data loading process of Paddle is much slower.
Before running the code, invoking ``paddle.device.get_device()`` and ``paddle.device.cuda.device_count()`` might be helpful to locate the problem.

Reply:
Thanks for the reply. We are using cpus if there are no heavy computation tasks. It is ok to leave it unchanged.